src/ray/protobuf/core_worker.proto

```diff
@@ -190,6 +190,8 @@ message CancelTaskRequest {
   bytes intended_task_id = 1;
   // Whether to kill the worker.
   bool force_kill = 2;
+  // Object ID of the remote task that should be killed.
+  bytes remote_object_id = 3;
```

Comment:
I chose to include this in the existing Cancelation RPC service to avoid the clutter of defining a separate service for a marginally different use case.

Reply:
I think it's actually a bit confusing to keep these in the same RPC type since the other fields don't really make sense anymore. Can you split this out into a separate RPC handler?

Reply:
Sure!